stdlib/2/__builtin__.pyi

```diff
@@ -943,7 +947,10 @@ class list(MutableSequence[_T], Generic[_T]):
     def remove(self, __value: _T) -> None: ...
     def reverse(self) -> None: ...
     if sys.version_info >= (3,):
-        def sort(self, *, key: Optional[Callable[[_T], Any]] = ..., reverse: bool = ...) -> None: ...
+        @overload
+        def sort(self: List[_SupportsLessThan], *, key: None = ..., reverse: bool = ...) -> None: ...
```

Comment:
This must actually be `List[_LT]` to avoid the invariance issue mentioned in my comment.